tensorflow_probability/python/bijectors/real_nvp_test.py

```diff
@@ -97,6 +97,46 @@ def testBatchedBijectorWithMLPTransform(self):
       self.assertAllClose(x_, inverse_y_, rtol=1e-1, atol=0.)
       self.assertAllClose(ildj_, -fldj_, rtol=1e-6, atol=0.)
 
+  def testBijectorConditionKwargs(self):
+    N = 3
+    x_ = np.arange(N * 4 * 2).astype(np.float32).reshape(N, 4 * 2)
+    conditions = {
+        'a': tf.reshape(tf.range(N * 4, dtype=tf.float32), (N, 4)),
+        'b': tf.reshape(tf.range(N * 2, dtype=tf.float32), (N, 2)),
+    }
+
+    with self.test_session() as sess:
+      nvp = tfb.RealNVP(
+          num_masked=4, validate_args=True, **self._real_nvp_kwargs)
+      x = tf.constant(x_)
+
+      forward_x = nvp.forward(x, **conditions)
+      # Use identity to invalidate cache.
+      inverse_y = nvp.inverse(tf.identity(forward_x), **conditions)
+      forward_inverse_y = nvp.forward(inverse_y, **conditions)
+      fldj = nvp.forward_log_det_jacobian(x, event_ndims=1, **conditions)
+      # Use identity to invalidate cache.
+      ildj = nvp.inverse_log_det_jacobian(
+          tf.identity(forward_x), event_ndims=1, **conditions)
+      tf.global_variables_initializer().run()
+      [
+          forward_x_,
+          inverse_y_,
+          forward_inverse_y_,
+          ildj_,
+          fldj_,
+      ] = sess.run([
+          forward_x,
+          inverse_y,
+          forward_inverse_y,
+          ildj,
+          fldj,
+      ])
+      self.assertEqual("real_nvp", nvp.name)
+      self.assertAllClose(forward_x_, forward_inverse_y_, rtol=1e-1, atol=0.)
```

Comment:
Can these tolerances be reduced? 1e-1 seems quite high.

Reply:
Changed. It seems like I just blindly copied the tolerances over from the other tests in the same file. I'm curious why they have so loose tolerances?